google-account-plugin/src/com/google/cloud/tools/intellij/stats/TrackingEventBuilder.java

```diff
@@ -39,19 +38,26 @@
   }
 
   @Override
-  public FluentTrackingEventWithValue withLabel(String label) {
+  public FluentTrackingEventWithMetadata withLabel(String label) {
     this.label = Preconditions.checkNotNull(label);
     return this;
   }
 
   @Override
-  public PingsAnalytics setValue(Integer value) {
+  public FluentTrackingEventWithMetadata withLabel(String label, int value) {
```

Comment:
And for this to work, you will store a list of label-value pairs so that clients can call `withLabel()` multiple times.